src/ape/managers/chain.py

```diff
@@ -136,12 +137,14 @@ def query(
             step=step,
             engine_to_use=engine_to_use,
         )
-
-        return self.query_manager.query(query)
+        vals = self.query_manager.query(query)
```

Comment:
Should probably flop this back to using `self.range(...)` and then converting to dataframe